EventFilter/SiPixelRawToDigi/plugins/RawToDigiGPU.cu

```diff
@@ -8,6 +8,9 @@
  *
 **/
 
+// local includes should go first....
```

Comment:
I guess there are two different philosophies :-)

Reply:
The idea is that each header should compile independently so they should be included in inverse order of dependency so that compiler will catch missing headers...
(ok, you may say we should verify this in unit tests, not in each compilation unit)